### DISCUSSION TITLE
Unignore certain files

### DISCUSSION THREAD

**Discussion by moltar** [vercel/turborepo#7612] [Help] [answered]
### Summary

Is it possible to un-ignore certain patterns from the `output` config?

E.g. I have `*.tgz` files in `.gitignore` but they are artifacts of `npm pack`.

I'd like to be able to cache this command as well.

But I think turbo is ignoring these files entirely, because they are in `.gitignore`.

Is there any way to undo this and force turbo to consider `*.tgz` files?

**Comment by anthonyshew** [comment, marked as answer]
I'm not sure if you are meaning inputs or outputs to a task, but yes to both:
- For inputs, we have the `$TURBO_DEFAULT$` microsyntax documented here: https://turbo.build/repo/docs/reference/configuration#inputs
- For outputs, you can use negative globs: https://turbo.build/repo/docs/reference/configuration#glob-specification-for-paths

**Reply by moltar** [reply]
> I'm not sure if you are meaning inputs or outputs to a task, but yes to both:

It is for `output`.

> negative globs

Hm, 🤔 the docs state:

- `!dist` - Ignore the dist directory and all of its contents
- `!dist/*.js` - Ignore all .js files in the dist directory

Doesn't sound like what I need.

I need the opposite.

UN-ignore files from the directory.
